src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/Kmer.java

```diff
@@ -67,7 +67,22 @@ public Kmer(final byte[] bases, final int start, final int length) {
         this.bases = bases;
         this.start = start;
         this.length = length;
-        this.hash = new String(bases, start, length).hashCode();
+        this.hash = hashCode(bases, start, length);
+    }
+
+    /**
+     *  Compute the hashcode of this KMer
```

Comment:
Since it's static it's not technically "computing the hashCode for **this** KMer" .